specs/windows/windows_eventlog.table

```diff
@@ -7,6 +7,7 @@ schema([
     Column("level", INTEGER, "Severity level associated with the event"),
     Column("provider_name", TEXT, "Provider name of the event"),
     Column("provider_guid", TEXT, "Provider guid of the event"),
+    Column("computer_name", TEXT, "Hostname of system where event was generated"),
```

Comment:
Updated as part of PR "[Adds support for the computer field in Windows Eventlogs
#6952](https://github.com/osquery/osquery/pull/6952/files#diff-dd760b19aa7115cbd9a4bbe4b85efffdff2097c683483a2090bb59b3a3e42229R10)